jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClient.java

```diff
@@ -300,7 +306,27 @@ public void reconnectionFailed(Exception e)
         {
             logger.debug("About to connect and login.");
         }
-        xmppConnection.connect().login();
+
+        try
+        {
+            xmppConnection.connect();
```

Comment:
actually this call should be wrapped in the retry task already, similar to what happens here: https://github.com/jitsi/jicoco/blob/master/jicoco/src/main/java/org/jitsi/meet/ComponentMain.java#L222

Reply:
This is the retry task? What do you mean?

Reply:
I mean that you're calling the connect twice, both here and in the task itself, also try catch block is duplicated

Reply:
Why do we retry the xmpp connection? Smack already does this

Reply:
Nope, it does it once we are connected. This retries the connection.

Reply:
it's to retry the initial connect attempt